docker/opensource-docker/scripts/hadoop-env.sh

```diff
@@ -1,3 +1,4 @@
+#!/bin/bash
```

Comment:
thanks for reminding me, I will remove this "#!bin/bash" lines from files in which this line doesn't appear originally.